src/Symfony/Component/Messenger/Transport/Doctrine/Connection.php

```diff
@@ -306,20 +314,20 @@ private function getSchema(): Schema
     {
         $schema = new Schema([], [], $this->driverConnection->getSchemaManager()->createSchemaConfig());
         $table = $schema->createTable($this->configuration['table_name']);
-        $table->addColumn('id', Type::BIGINT)
+        $table->addColumn('id', Types::BIGINT)
```

Comment:
you cannot require `Types` as it's not released in dbal yet. please just keep using `Type` everywhere. we can handle the deprecation of it in the future when it's actually released.

Reply:
You're right!
Would like a method equivalent with checking if the class exist?

Reply:
only use Type in this PR please. feel free to create a second PR with a solution for the future deprecation

Reply:
It's still not consistent. Here you implemented the fallback https://github.com/symfony/symfony/blob/1e04345fd53b577141f57fe9df25ab9b6a249513/src/Symfony/Component/Messenger/Transport/Doctrine/Connection.php#L346-L352 but https://github.com/symfony/symfony/blob/1e04345fd53b577141f57fe9df25ab9b6a249513/src/Symfony/Component/Messenger/Transport/Doctrine/Connection.php#L325 does not use it. Either use the `getDateTimeType` helper everywhere, or don't add it at all.

Reply:
I asked to not use the new `Types` class because in order to fully fix the deprcation of `Type`, you would also need to add the fallback of the other types like "text" in https://github.com/symfony/symfony/blob/1e04345fd53b577141f57fe9df25ab9b6a249513/src/Symfony/Component/Messenger/Transport/Doctrine/Connection.php#L321 which you did not. So it's not complete and irrelevant to this PR.